css/main.css

```diff
@@ -24,13 +19,37 @@ html, body {
   }
 }
 
-/* lg */
 @media (min-width: 1200px) {
   html {
     font-size: 14px;
   }
 }
 
+/* lg */
+@media (min-width: 1920px) {
```

Comment:
The documentation says the `.lg-*` breakpoint is `1200px`, but in our `bootstrap.min.css` it's actually `1920px`.